### DISCUSSION TITLE
Pause and Resume of a command line ASReview simulation

### DISCUSSION THREAD

**Discussion by zhaokoh** [asreview/asreview#1315] [Q&A] [answered]
Hello all,

I am exploring the command line simulation in ASReview and I am wondering if there is any way to:

1. Pause a long-running simulation (by saving the current progress - perhaps into a state file) then resume at a later stage?

2. Or interrupt (gently kill) the simulation process but force the simulation to write the state file so that I can perform some interim analysis with the state file?

The reason I am asking is that I have 90K records, it is now up to 60K labelled records after almost 1.5 days, and it is up to 99% relevant data. I didn't set a max and I did not want to re-run the process again.

Any suggestion or pointer would be much appreciated 🙏 Thanks.

**Comment by J535D165** [comment, marked as answer]
Dear @zhaokoh, there is no option to pause the simulation at the moment. You have to wait till the last relevant record is found. So hopefully you will find your last relevant record soon. 

I will add this as a feature request to our backlog.

**Reply by zhaokoh** [reply]
Thanks @J535D165
